Releases.md

```diff
@@ -402,7 +402,7 @@
 - docs(flags): fix broken link to minimist (#2842)
 - docs(fs): remove misleading docs from fs.walk (#2836)
 - docs(log): add note for module authors (#2843)
-- refactor: cleanup check licence headers tool (#2830)
+- refactor: cleanup check license headers tool (#2830)
```

Comment:
This change appears to be a mistake. [Licence](https://www.macquariedictionary.com.au/features/word/search/?word=licence&search_word_type=Dictionary) is the noun and [license](https://www.macquariedictionary.com.au/features/word/search/?word=license&search_word_type=Dictionary) is the verb.
```suggestion
- refactor: cleanup check licence headers tool (#2830)
```